share/completions/ln.fish

```diff
@@ -1,16 +1,16 @@
+complete -c ln -s s -l symbolic            -d 'Make symbolic links instead of hard links'
+complete -c ln      -l backup              -d 'Make a backup of each existing destination file' -a "none off numbered t existing nil simple never"
+complete -c ln -s b                        -d 'Make a backup of each existing destination file'
+complete -c ln -s dF -l directory          -d 'Allow superuser to attempt to hard link directories'
```

Comment:
Note that more than a single letter short option is not documented to work. From the `complete` documentation:

> SHORT_OPTION is a one character option for the command.

And if you press \<tab\> after typing `-` then try to select that you get just `-d`. On the other hand if you type `-d` and press \<tab\> it adds the `F`. Not sure if both behaviors are intentional, accidental, or an error but we probably don't want to rely on it. If you want to support `-F` as an alias for `-s` you need a separate completion.